llvm/test/MC/AArch64/global-tagging.ll

```diff
@@ -9,9 +9,12 @@
 ; RUN: llvm-mc -filetype=obj %t.S -triple=aarch64-linux-android31 -o %t.o
 ; RUN: llvm-readelf -r %t.o | FileCheck %s --check-prefix=CHECK-RELOCS
 
-; RUN: obj2yaml %t.o -o %t.yaml
-; RUN: FileCheck %s --input-file=%t.yaml --check-prefix=CHECK-YAML
+; RUN: obj2yaml %t.o -o %t.o.yaml
+; RUN: FileCheck %s --input-file=%t.o.yaml --check-prefix=CHECK-OYAML
 ; RUN: llvm-readelf -r %t.o | FileCheck %s --check-prefix=CHECK-RELOCS
+; RUN: ld.lld %t.o -o %t.so
```

Comment:
I don't think LLD is built as part of `check-llvm` so I would expect this to fail unless a new-enough LLD happens to be on `$PATH`. The new test probably needs to be added to LLD instead.